en/plugins/overview-plugins.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Indigo.Design Plugins
+_description: The Indigo.Design Plugins for Figma, Sketch and XD allow you to quickly turn your design to a full app inside the Indigo Design App Builder.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Figma, AdobeXD, Ignite UI for Angular, Sketch to Angular, Figma to Angular, XD to Angular, Angular, Angular Design System, Export code from Sketch, Export code from Figma, Export code from XD, Design Kits for Angular, Sketch HTML, Figma to HTML, XD to HTML, Sketch UI kits, Figma UI kits, XD UI kits, Sketch Theme, Theme Editor
+---
+
+# Overview
+
+The Indigo.Design Plugins for Figma, Sketch and XD allow you to quickly turn your design to a full app inside the Indigo Design App Builder.
+
+Take your designs and create interactive, responsive apps with real UI components and styling. Use the `Create New App` feature of the plugins to transform your Figma, Sketch and Adobe XD designs into a real applications under the AppBuilder, allowing access to the images and themes - all of this directly in the cloud.
+
+The plugins allow you to choose a name and resolution for your app, as well as to select the Indigo.Design Cloud workspace, where the app will be created.
+
+<img class="responsive-img" src="../images/create-from-common.png" />
+
```

Comment:
How about adding a note here, that users should have Infragistics account and Indigo.Design subscription or start a trial in order to import apps in AppBuilder. Also a link to the subscription plans - https://www.infragistics.com/products/indigo-design/pricing